workflows/quickstart/index.ts

```diff
@@ -35,12 +35,13 @@ const client: ExecutionsClient = new ExecutionsClient();
  * @param {string} projectId The Google Cloud Project containing the workflow
  * @param {string} location The workflow location
  * @param {string} workflow The workflow name
- * @param {string} searchTerm Optional search term to pass as runtime argument to Workflow
+ * @param {string} searchTerm Optional search term to pass to the Workflow as a runtime argument
  */
 async function executeWorkflow(
   projectId: string,
   location: string,
-  workflow: string
+  workflow: string,
+  searchTerm = ''
```

Comment:
When you compile this typescript, this should be throwing a typing error at you since you are haven't given a type to `searchTerm` yet.

Reply:
Would like to highlight this again. Please take a gander at the typing for the function. ☝🏼

Reply:
I changed this but i got clobbered

Reply:
Good to know 👍🏼  - if you can add back in `:string` and set the default manually, that should not get undone by owlbot.